Added usecase process graphs (#4)
Open-EO/openeo-community-examples: python/README.md

# Python Client - Examples and Code Snippets

This folder contains code snippets for the openEO Python client.

## Overview

Environments: `Python` (plain Python code), `Jupyter` (e.g. Notebooks)

> **Furthermore in usecase_notebooks repository:**

Reviewer: This needs to be removed.

Author: updated

| Title | Environment | Description |
| ----- | ----------- | ----------- |

> | BurntMapping   | `Jupyter`    | Classical Normalized Burnt Ratio(NBR) difference performed using VITO backend on a chunk polygon. The method followed in this notebook to compute DNBR is inspired from [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping). |

Reviewer: Please add links to the individual folders:

```suggestion
| [BurntMapping](./BurntMapping/)   | `Jupyter`    | Classical Normalized Burnt Ratio(NBR) difference performed using VITO backend on a chunk polygon. The method followed in this notebook to compute DNBR is inspired from [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping). |
```

Author: update

> | FloodNDWI   | `Jupyter`    | Comparative study between pre and post image for Colong during 2021 flood using NDWI. [Refernce](https://labo.obs-mip.fr/multitemp/the-ndwi-applied-to-the-recent-flooding-in-the-central-us/) |

Reviewer: ```suggestion
| FloodNDWI   | `Jupyter`    | Comparative study between pre and post image for Colone during 2021 flood using NDWI. [Refernce](https://labo.obs-mip.fr/multitemp/the-ndwi-applied-to-the-recent-flooding-in-the-central-us/) |
```

Author: updated

> | FloodSAR   | `Jupyter`    | Flood extent detection following [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping) using openeo process.|

Reviewer: ```suggestion
| [FloodSAR](./FloodSAR/)   | `Jupyter`    | Flood extent detection following [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping). |
```

| flood_ndwi   | `Jupyter`    | Comparative study between pre and post image for Colong during 2021 flood using NDWI. [Refernce](https://labo.obs-mip.fr/multitemp/the-ndwi-applied-to-the-recent-flooding-in-the-central-us/) |
| flood_sar_udf   | `Jupyter`    | Flood extent detection following [UN SPIDER's recommended practices](https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping) using openeo process.|
| ndwi_basic   | `Jupyter`    | Executes simple process like ndwi already available in EOplaza. Service available [here](https://portal.terrascope.be/catalogue/app-details/13). |
| rescale_chunks   | `Jupyter`    | The creation of a simple process to rescale Sentinel 2 RGB image along with the use of chunk_polygon apply with a (User Defined Function) UDF. |

## Contributing

> * OpenEO/EOplaza users can also send a pull request in this repository as either a python file/notebook for their usecase. Showcasing their usecases are highly encouraged.

Reviewer: ```suggestion
* OpenEO users can send a pull request to this repository as either a Python file or a Jupyter notebook for their usecase. Submissions should be in separate folders. Showcasing their usecases are highly encouraged.
```

Environments: `Python` (plain Python code), `Jupyter` (e.g. Notebooks)

| Title | Environment | Description |
| ----- | ----------- | ----------- |

> | [Anomaly_Detection](./RescaleChunks/)   | `Python`  | Check the crop growth on your field and compare it with similar fields in the region.|

Reviewer: These are all Python Notebooks as far as I can see, so the Environment should be set to `Jupyter` instead of `Python`
